docs/content/preview/releases/ybdb-releases/v2.23.md

```diff
@@ -0,0 +1,426 @@
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-darwin-x86_64.tar.gz">
+      <i class="fa-brands fa-apple"></i>
+      <span>macOS</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-linux-x86_64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux x86</span>
+    </a>
+  </li>
+  <li>
+    <a href="https://downloads.yugabyte.com/releases/2.23.0.0/yugabyte-2.23.0.0-b710-el8-aarch64.tar.gz">
+      <i class="fa-brands fa-linux"></i>
+      <span>Linux ARM</span>
+    </a>
+  </li>
+</ul>
+
+**Docker:**
+
+```sh
+docker pull yugabytedb/yugabyte:2.23.0.0-b710
+```
+
+### Highlights
+
+We're excited to announce the [technical preview](/preview/releases/versioning/#tech-preview-tp) of four powerful new features in YugabyteDB aimed at simplifying operations and enhancing functionality.
+
+**Database instant cloning** {{<badge/tp>}}
+
+[Database Cloning](/preview/manage/backup-restore/lightweight-db-clone/) allows you to quickly create independent copies of your database for data recovery, development, and testing. Clones can be created in seconds, and initially consume no additional disk space, because they share the same data files as the original database at creation but operate independently thereafter. Clones can be created as of now, or as of any time in the recent past (within a configurable history retention period), enabling developers to experiment without impacting production performance. This feature also provides a safety net for rapid recovery from accidental data loss or corruption.
+
+**pg_cron extension** {{<badge/tp>}}
+
+We're introducing support for the [pg_cron extension](/preview/explore/ysql-language-features/pg-extensions/extension-pgcron/), which brings a cron-based job scheduler directly into the database. With pg_cron, you can schedule YSQL commands using familiar cron syntax, including jobs on intervals as fine as seconds. While pg_cron operates on a single node known as the pg_cron leader, the queries it schedules leverage the full resources of the distributed cluster. In case of node failure, leadership is automatically transferred, ensuring continuous availability and smooth scheduling operations.
+
+**DB Scoped xCluster replication** {{<badge/tp>}}
+
+[DB Scoped xCluster replication](/preview/deploy/multi-dc/async-replication/async-replication-transactional/) simplifies the management of YSQL transactional xCluster replication by operating at the database level instead of the table level. This reduces the need for IT/admin involvement when tables are created or dropped, as xCluster management is only required for adding or removing entire databases from replication. DDL operations can be performed by authorized users or DBAs without needing elevated privileges. DB Scoped xCluster supports YSQL transactional replication and is ideal for Disaster Recovery (DR) purposes.
```

Comment:
```suggestion
[Semi-automatic xCluster replication](/preview/deploy/multi-dc/async-replication/async-replication-transactional/) simplifies the management of YSQL transactional xCluster replication by operating at the database level instead of the table level. This reduces the need for IT admin involvement when tables are created or dropped, as xCluster management is only required for adding or removing entire databases from replication. DDL operations can be performed by authorized users or DBAs without needing elevated privileges. Semi-automatic xCluster supports YSQL transactional replication and is ideal for Disaster Recovery (DR) purposes.
```